airbyte-config-oss/init-oss/src/main/resources/seed/source_definitions.yaml

```diff
@@ -2006,7 +2006,7 @@
 - name: Stripe
   sourceDefinitionId: e094cb9a-26de-4645-8761-65c0c425d1de
   dockerRepository: airbyte/source-stripe
-  dockerImageTag: 3.0.5
+  dockerImageTag: 3.1.0
```

Comment:
3.0.4

Reply:
@bazarnov we're following the guidelines in contributing to airbyte made by the connector team https://docs.airbyte.com/contributing-to-airbyte/#examples